update.py

```
@@ -1 +1,135 @@
+
+    return should_update, version
+
+
+def get_version():
+    """ Check the most recent version of the script. """
+    response = requests.get(version_url, timeout=10)
+
+    if response.status_code == 200:
+        return response.json()[0]['sha']
+    else:
+        raise RuntimeError("Unable to find the most recent version of the script.")
```

Ruff (0.13.3)

Avoid specifying long messages outside the exception class

(TRY003)